src/main.ts

```diff
@@ -215,7 +220,7 @@ export default class RecipeGrabber extends Plugin {
 				// magicTime datePublished
 			} else if (arguments.length == 3) {
 				// magicTime datePublished "dd-mm-yyyy HH:MM"
-				if (new Date(arg1) == "Invalid Date") {
+				if (isValidDate(arg1)) {
```

Comment:
```suggestion
				if (!isValidDate(arg1)) {
```
If it's not a valid date, return an error.

Reply:
Ha. Good catch!